src/crypto/sha256.cpp

```diff
@@ -29,6 +29,15 @@ namespace sha256d64_avx2
 void Transform_8way(unsigned char* out, const unsigned char* in);
 }
 
+#if defined(__linux__) && defined(HAVE_POWER8)
+#include <sys/auxv.h>
```

Comment:
I don't think we need this here, and POWER vectoring stuff seems to screw with standard C++ stuff, so maybe not a good idea to have it.

Reply:
auxv isnt POWER- or vector-specific, its to get getauxval which is a libc function.